### PR TITLE
Iana/tests refactor

### PR DESCRIPTION
**Pull request by alekseevaiana**
- add tests;
- make max-width;
- update readme;
- add autocomplete for add ingredients;

### CODE REVIEW

File: src/__tests__/AddIngridient.test.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import AddIngridient from "../components/AddIngridient";
+import userEvent from "@testing-library/user-event";
+
+describe("AddIngridients", () => {
+  test("Input for ingridient works", async () => {
```
Comment (slonoed) [inline src/__tests__/AddIngridient.test.js]: I don't think this test make sense. You are testing 3rd party library input standard functionality.

File: src/__tests__/AddIngridient.test.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      <AddIngridient allIngridients={allIngridients} />
+    );
+    const input = getByRole("textbox");
+    userEvent.type(input, "rice");
+    expect(input).toHaveValue("rice");
+  });
+
+  test("Ingridient should be added", async () => {
+    const onChange = jest.fn();
+    const allIngridients = ["ground beef", "pasta", "meat"];
```
Comment (slonoed) [inline src/__tests__/AddIngridient.test.js:9]: This variable is not used

File: src/__tests__/CreateMeal.test.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import CreateMeal from "../components/CreateMeal";
+
+describe("CreateMeal", () => {
+  test("renders create meal component", async () => {
+    const allIngridients = ["ground beef", "pasta", "meat"];
+    render(<CreateMeal allIngridients={allIngridients} />);
```
Comment (slonoed) [inline src/__tests__/CreateMeal.test.js:8]: The only thing you test here is that there are no exceptions in render. Is it what you want? If "yes" then change name to make it more obvious.

File: src/__tests__/IngridientsList.test.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    const list = ["cheese", "lemon"];
+    const { queryByText } = render(<IngridientsList ingridients={list} />);
+
+    expect(queryByText("cheese")).not.toEqual(null);
+    expect(queryByText("lemon")).not.toEqual(null);
+  });
+});
+
+describe("IngridientsList", () => {
+  test("Delete ingridient", async () => {
```
Comment (slonoed) [inline src/__tests__/IngridientsList.test.js]: Nit. I would suggest more descriptive naming: "Should call `handleDelete` when user click clear icon"

File: src/__tests__/MealForm.test.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+import userEvent from "@testing-library/user-event";
+import MealForm from "../components/MealForm";
+
+describe("MealForm", () => {
+  test("render name in input", async () => {
+    const allIngridients = ["ground beef", "pasta", "meat"];
+
+    const mealName = "potato";
+    const value = { name: mealName, ingridients: [] };
+    console.log("MEAL FORM ALLINGRIDIENTS =====>>", allIngridients);
```
Comment (slonoed) [inline src/__tests__/MealForm.test.js]: Log

File: src/components/Groceries.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import { Box, Typography } from "@mui/material";
+
+export default function Groceries() {
+  return (
+    <Box>
+      <Typography variant="h6" color="primary.dark">
+        Groceries
+      </Typography>
+      {/* 
+      - Render checklist
```
Comment (slonoed) [inline src/components/Groceries.js:10]: Remove if not needed.

File: src/__tests__/AddIngridient.test.js (hunk 2 above)
Comment (alekseevaiana) [reply src/__tests__/AddIngridient.test.js:9]: Used as a parameter

File: src/__tests__/IngridientsList.test.js (hunk 4 above)
Comment (alekseevaiana) [reply src/__tests__/IngridientsList.test.js]: Ok, thanks!

**Pull request merged**